scipy/sparse/linalg/isolve/tests/test_lsmr.py

```diff
@@ -82,7 +82,7 @@ def testInitialization(self):
         # Test warm-start with single iteration
         x0 = lsmr(G, b, maxiter=1)[0]
         x = lsmr(G, b, x0=x0)[0]
-        assert_array_almost_equal(x_ref, x)
+        assert_array_almost_equal(x_ref, x, decimal=5)
```

Comment:
here and elsewhere for some of the patches I pulled in from @jjhelmus on the feedstock, we could also add the `platform.machine()` guards I've used elsewhere, to retain the stringency on other architectures.